btc-lsp/dhall/regtest/k8s.lsp.dhall

```diff
@@ -11,7 +11,8 @@ let network = G.BitcoinNetwork.RegTest
 in  { apiVersion = "v1"
     , kind = "List"
     , items =
-      [ K.Resource.Service (Lsp.mkService network)
+      [ K.Resource.Service
+          (Lsp.mkService network (Some C.ProviderType.DigitalOcean))
```

Comment:
on regtest there should be no cloudProvider (None C.ProviderType)

Reply:
done